features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -92,10 +99,12 @@ internal class SessionReplayFeature(
         }
 
         this.appContext = appContext
-
         sdkCore.setEventReceiver(SESSION_REPLAY_FEATURE_NAME, this)
+        if (RESOURCES_ENDPOINT_ENABLED) {
+            registerResourceFeature(sdkCore)
+        }
         dataWriter = createDataWriter()
-        sessionReplayRecorder = sessionReplayRecorderProvider(dataWriter, appContext)
+        sessionReplayRecorder = sessionReplayRecorderProvider(resourcesWriter, dataWriter, appContext)
```

Comment:
I believe it should be `resourceFeature.resourcesWriter` here and then `SessionReplayFeature#resourcesWriter` is not needed here (I guess it is just temporary)?

```suggestion
        sessionReplayRecorder = sessionReplayRecorderProvider(resourceFeature.resourcesWriter, dataWriter, appContext)
```